packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -83,20 +106,20 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
           ref={ this._resolveRef('_checkBox') }
           id={ this._id }
           name={ name || this._id }
-          className={ css('ms-Checkbox-input', styles.input) }
+          className={ classNames.input }
           type='checkbox'
           onChange={ this._onChange }
           onFocus={ this._onFocus }
           onBlur={ this._onBlur }
           aria-checked={ isChecked }
         />
-        <label className={ css('ms-Checkbox-label', styles.label) } htmlFor={ this._id }>
-          <span className={ css('ms-Checkbox-box', styles.box) }>
-            <span className={ css('ms-Checkbox-checkbox', styles.checkbox) }>
-              <Icon iconName='CheckMark' className={ styles.checkmark } />
+        <label className={ classNames.label } htmlFor={ this._id } >
+          <span className={ classNames.box }>
```

Comment:
these should be divs (not that it matters that much). we use divs for blocks and inline-blocks, and spans for inlines